src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/ActionsRow.js

```diff
@@ -17,10 +17,21 @@ export const ActionsRow = ({ option, actions, onChange }) => {
   return actions.map((action, actionIndex) => {
     if (!action) return null;
 
+    const collator = new Intl.Collator();
```

Comment:
Please move this sorting logic to some utils files

Reply:
done